enginetest/script_queries.go

```diff
@@ -700,6 +700,34 @@ var ScriptTests = []ScriptTest{
 			{3, 3},
 		},
 	},
+	{
+		Name: "Indexed Join On Keyless Table",
```

Comment:
@zachmu What's the right way to skip this test in dolt, since we don't support indexes on keyless tables? Or should I just leave this test out for now?

Reply:
Looks like we can skip the whole thing dolt-side with:

```diff
diff --git a/go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go b/go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go
index 58bca274e..5ee14a568 100644
--- a/go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go
+++ b/go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go
@@ -259,11 +259,14 @@ func TestTruncate(t *testing.T) {
 }
 
 func TestScripts(t *testing.T) {
+       skipped := []string{
+               "create index r_c0 on r (c0);",
+       }
        t.Run("no transactions", func(t *testing.T) {
-               enginetest.TestScripts(t, newDoltHarness(t))
+               enginetest.TestScripts(t, newDoltHarness(t).WithSkippedQueries(skipped))
        })
        t.Run("with transactions", func(t *testing.T) {
-               enginetest.TestScripts(t, newDoltHarness(t).withTransactionsEnabled(true))
+               enginetest.TestScripts(t, newDoltHarness(t).withTransactionsEnabled(true).WithSkippedQueries(skipped))
        })
 }

```

Reply:
Yup exactly